services/stackexchange/stackexchange-reputation.service.js

```diff
@@ -0,0 +1,82 @@
+'use strict'
+
+const BaseJsonService = require('../base-json')
+const { metric } = require('../../lib/text-formatters')
+const { floorCount: floorCountColor } = require('../../lib/color-formatters')
+const Joi = require('joi')
+
+const reputationSchema = Joi.object({
+  items: Joi.array()
+    .min(1)
```

Comment:
For this particular API call, there's always a single value returned unless the query provided by the client doesn't exist in StackExchange and therefore return an empty array.

I guess I can make an error case for if I get an empty array back from StackExchange,
I output a badge with "0" as its value? Then change the schema so that its min(0), max(1)

Reply:
Okay. Yea, you can use `.length(1)`, then. I don't think another test case is needed; when the schema is broken we can trust the Joi will do the right thing.